mmv1/products/monitoring/MonitoredProject.yaml

```diff
@@ -24,9 +24,6 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     'Official Documentation': 'https://cloud.google.com/monitoring/settings/manage-api'
   api: 'https://cloud.google.com/monitoring/api/ref_v3/rest/v1/locations.global.metricsScopes.projects'
```

Comment:
Line 18 in this file. Delete url to **v1/{{name}}**

Reply:
Thanks! Is there an explanation for why this change is now necessary?

For posterity, here's the conversation that I think led to this recommendation:

> I think the problem comes down to whatever is decoding the deletion. In the latest CI run of `TestAccMonitoringMonitoredProject_monitoringMonitoredProjectBasicExample`, the deletion request is 

> `DELETE /v1/locations/global/metricsScopes/ci-test-project-188019/projects/locations/global/metricsScopes/1067888929963/projects/102753631080`

> and we can see the path has the format `v1/locations/global/metricsScopes/<project>/projects/<metric_scope_path>`

> when we just want `DELETE /v1/<metric_scope_path>` which would translate to `DELETE /v1/locations/global/metricsScopes/1067888929963/projects/10275363108`. I added debug lines to the decoder, and the decoder seems to decode the link just fine.

> This leads me to suspect some spooky complexity from deleting the `nested_query` functionality, in which case I think I definitely need more support in debugging

Reply:
It's mainly due to to deleting the `nested_query`. I don't know why it was implemented with `nested_query` before. Let the CI runs complete and see if the proposed solution solved the problem for all failing tests.